src/index.d.ts

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+declare namespace WixAnimations {
+  type Direction = 'top' | 'bottom' | 'left' | 'right';
+  type Sequence = 'default' | 'flip' | 'reverse' | 'reverse-flip';
+  type Timing = 'micro' | 'small' | 'medium' | 'large';
+
+  interface Delay {
+    enter?: number;
+    exit?: number;
+  }
+
+  interface Segment {
+    direction?: Direction;
+    size?: string;
+  }
+
+  interface Translate {
+    enter?: Direction | Segment;
+    exit?: Direction | Segment;
+  }
+
+  interface AnimatorProps {
+    delay?: Delay;
+    timing?: Timing;
+    opacity?: boolean;
+    scale?: boolean;
+    translate?: boolean | string | Translate;
+    height?: boolean | number | ((element) => number);
+    width?: boolean | number | ((element) => number);
+    sequence?: Sequence;
```

Comment:
I see you don't accept the same types as in the prop types
for example - `sequence`
https://github.com/wix/wix-animations/blob/f7720c3377014a027c84ee081d5c443a306a7a01/src/components/Animator/components/AnimatorParent.js#L35
Be aware that if TS consumers of this library are still passing the old types, their build will break once your change is published.

it happens in other types here as well.